samples/nrf9160/http_update/full_modem_update/README.rst

```diff
@@ -7,21 +7,24 @@ nRF9160: HTTP full modem update
    :local:
    :depth: 2
 
-The HTTP full modem update sample demonstrates how to perform a full firmware update of the modem (as opposed to a delta update).
-The sample downloads a modem firmware signed by Nordic and performs the firmware update of the modem.
+The HTTP full modem update sample shows how to perform a full firmware update of the modem.
+The sample downloads a modem firmware signed by Nordic Semiconductor and then performs the firmware update of the modem.
 
 Overview
 ********
 
 An external flash memory with a minimum of 2MB of free space is required to perform a full modem update.
 Hence, only versions 0.14.0 and later of the nrf9160 DK support this sample as the prior versions do not have any external flash memory.
 
-The sample connects to a remote HTTP server to download a signed version of the modem firmware, using the :ref:`lib_fota_download` library.
-It then writes the firmware to the external flash memory.
-After the modem firmware has been stored in the external flash memory, the sample will use the :ref:`lib_fmfu_fdev` to pre-validate the update and program it to the modem.
+The sample proceeds as follows:
 
-Two different versions can be downloaded, depending on what version is currently installed.
-The version which is not currently installed is selected.
+* It connects to a remote HTTP server to download a signed version of the modem firmware, using the :ref:`lib_fota_download` library.
+* It writes the modem firmware to the external flash memory.
+* It prevalidates the update if the firmware supports the prevalidation process.
+* It then programs the update to the modem, using the :ref:`lib_fmfu_fdev` library.
+
+The current version of this sample downloads two different versions of the firmware, namely 1.2.1 and 1.2.2.
```

Comment:
I'm not sure it's a good idea to specifically mention the versions, it is not future proof and easy to forget to update this doc in case the sample is modified to download newer versions.

Reply:
@b-gent  one of the issues that came out was that this sample is not using the most updated version of the modem firmware.

Reply:
I see, ok